source/adapters/level_zero/platform.cpp

```diff
@@ -69,7 +70,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urPlatformGet(
   }
 
   if (ZeResult != ZE_RESULT_SUCCESS) {
-    urPrint("zeInit: Level Zero initialization failure\n");
+    logger::debug("zeInit: Level Zero initialization failure");
```

Comment:
Sounds like at least a warning message, not debug.

Reply:
Done